config.crc.env

```diff
@@ -0,0 +1,7 @@
+KUBERNETES_PROVIDER=crc
```

Comment:
instead of providing multiple contexts in different configs, why not have a way to pick up the current kubernetes context. And in the case of crc, minishift, minikube, openshift or kubernetes or any other kubernetes provider, the idea of context is the one would stay the same.